source/cloud-security/amazon/services/supported-services/amazon-security-lake/security-lake-source.rst

```diff
@@ -0,0 +1,343 @@
+    * AWS credentials for the account with permissions to write to the S3 bucket: ``AWS_ACCESS_KEY_ID`` and ``AWS_SECRET_ACCESS_KEY``.
+    * AWS S3 bucket details: ``AWS_REGION`` and ``S3_BUCKET`` (the S3 bucket name for raw events).
+
+#. Use the `Logstash keystore <https://www.elastic.co/guide/en/logstash/current/keystore.html>`__ to securely store these values.
+
+#. Create the configuration file ``indexer-to-s3.conf`` in the ``/etc/logstash/conf.d/`` folder:
+
+    .. code-block:: console
+
+        sudo touch /etc/logstash/conf.d/indexer-to-s3.conf
+
+#. Add the following configuration to the ``indexer-to-s3.conf`` file.
+
+    .. code-block:: ruby
+
+        input {
+            opensearch {
+                hosts =>  ["<WAZUH_INDEXER_ADDRESS>:9200"]
+                user  =>  "${INDEXER_USERNAME}"
+                password  =>  "${INDEXER_PASSWORD}"
+                ssl => true
+                ca_file => "</PATH/TO/WAZUH_INDEXER/CERTIFICATE>/root-ca.pem"
+                index =>  "wazuh-alerts-4.x-*"
+                query =>  '{
+                    "query": {
+                        "range": {
+                            "@timestamp": {
+                            "gt": "now-5m"
+                            }
+                        }
+                    }
+                }'
+                schedule => "*/5 * * * *"
+            }
+        }
+
+        output {
+            stdout {
+                id => "output.stdout"
+                codec => json_lines
+            }
+            s3 {
+                id => "output.s3"
+                access_key_id => "${AWS_ACCESS_KEY_ID}"
+                secret_access_key => "${AWS_SECRET_ACCESS_KEY}"
+                region => "${AWS_REGION}"
+                bucket => "${S3_BUCKET}"
+                codec => "json_lines"
+                retry_count => 0
+                validate_credentials_on_root_bucket => false
+                prefix => "%{+YYYY}%{+MM}%{+dd}"
+                server_side_encryption => true
+                server_side_encryption_algorithm => "AES256"
+                additional_settings => {
+                "force_path_style" => true
+                }
+                time_file => 5
+            }
+        }
+
+Running Logstash
+^^^^^^^^^^^^^^^^^
+
+#. Once you have everything set, run Logstash from the CLI with your configuration:
+
+    .. code-block:: console
+
+        sudo systemctl stop logstash
+        sudo -E /usr/share/logstash/bin/logstash -f /etc/logstash/conf.d/indexer-to-s3.conf --path.settings /etc/logstash ----config.test_and_exit
+
+#. After confirming that the configuration loads correctly without errors, run Logstash as a service.
+
+    .. code-block:: console
+
+        sudo systemctl enable logstash
+        sudo systemctl start logstash
+
+
+OCSF Mapping
+-------------
+
+The integration maps Wazuh Security Events to the **OCSF v1.1.0** `Security Finding (2001) <https://schema.ocsf.io/classes/security_finding>`__ Class.
```

Comment:
Use italics for for emphasis. For example `*OCSF v1.1.0*`. Use bold font style for UI elements.